docs/data/install-bestman-xrootd.md

```diff
@@ -0,0 +1,264 @@
+
+Requirements
+============
+
+Host and OS
+-----------
+
+-   OS is Red Hat Enterprise Linux 6, 7, and variants (see [details...](../common/yum.md))
+-   [EPEL](http://fedoraproject.org/wiki/EPEL) repos enabled.
+-   A working XRootD Server. See [InstallXrootd](https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/InstallXrootd) for details.
+-   Root access
+
+Users
+-----
+
+This installation will create several users unless they are already created.
+
+| User      | Comment                                                 |
+|:----------|:--------------------------------------------------------|
+| `bestman` | Used by Bestman SRM server (needs sudo access).         |
+| `daemon`  | Used by globus-gridftp-server.                          |
+| `xrootd`  | Used by the xrootd client to contact xrootd redirector. |
+
+For this package to function correctly, you will have to create the users needed for grid operation. Any user that can be authenticated should be created.
+
+For grid-mapfile users, each line of the grid-mapfile is a certificate/user pair. Each user in this file should be created on the server.
+
+For gums users, this means that each user that can be authenticated by gums should be created on the server.
+
+Note that these users must be kept in sync with the authentication method. For instance, if new users or rules are added in gums, then new users should also be added here.
+
+Certificates
+------------
+
+| Certificate                 | User that owns certificate | Path to certificate                                                                                 |
+|:----------------------------|:---------------------------|:----------------------------------------------------------------------------------------------------|
+| Host certificate            | `root`                     | `/etc/grid-security/hostcert.pem` `/etc/grid-security/hostkey.pem`                       |
+| Bestman service certificate | `bestman`                  | `/etc/grid-security/bestman/bestmancert.pem` `/etc/grid-security/bestman/bestmankey.pem` |
+
+[Instructions](../common/pki-cli.md) to request a service certificate.
+
+You will also need a copy of CA certificates (see below).
+
+Networking
+----------
+
+| Service Name            | Protocol    | Port Number                      | Inbound | Outbound |Commen                     |
+|:------------------------|:------------|:---------------------------------|:--------|:---------|:--------------------------|
+| GridFTP                 | tcp         |2811 and `GLOBUS_TCP_SOURCE_RANGE`| YES     |          | contiguous range of ports |
+| Storage Resource Manager| tcp         | 8080                             | YES     |          |                           |
```

Comment:
Nope, we shouldn't be listening on this.

Reply:
Fixed